src/App.tsx

```diff
@@ -15,71 +16,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+    isGoodSelected: true,
```

Comment:
The `isGoodSelected` state is redundant. You can determine if a good is selected by checking if `selectedGood` is an empty string or not.